### ISSUE TITLE
Webui vs config where does the configuration come from?

### ISSUE THREAD

**Issue by lDemonik** [vstakhov/rspamd#2146]
### Classification (Please choose *one* option):

* [ ] Crash/Hang/Data loss
* [x] WebUI/Usability
* [ ] Serious bug
* [ ] Other bug
* [ ] Feature
* [ ] Enhancement

### Reproducibility (Please choose *one* option):

* [X] Always
* [ ] Sometimes
* [ ] Rarely
* [ ] Unable
* [ ] I didn’t try
* [ ] Not applicable

### Additional information:
I am writing in the /override.d/account.conf
```
add_header = null;
reject = 12;
rewrite = 5;
subject = "*** SPAM ***% s";
greylist = 4;
```
I make rspamadm configdump actions
```
*** Section actions ***
add_header = null;
reject = 12;
rewrite = 5;
subject = "*** SPAM ***% s";
greylist = 4;

*** End of section actions ***
```
rspamadm configdump too correct
Then I reboot rspamd
And in WebUI still information
![default](https://user-images.githubusercontent.com/33311563/38292601-cca83a22-37fd-11e8-94de-85a802cd8ad4.jpg)
How can I view the current working configuration?
Rspamd 1.7.2.

**Comment by moisseev** [comment]
Configuration changes that you've made in WebUI are stored in `$DBDIR/rspamd_dynamic`.
The `rspamadm configdump` only shows static configuration (https://github.com/vstakhov/rspamd/issues/2135), but dynamic one has precedence over it.

**Comment by lDemonik** [comment]
And how can I say I can disable something that I do not need to do or change the configuration priority?

**Comment by stephdl** [comment]
>  Only delete rspamd_dynamic???
yes actually it is the way, remove the dynamic content. It was not the case (shout if I'm wrong) with the version 1.6.X, where you could overwrite the UI
